util/util.js

```diff
@@ -99,11 +99,19 @@ const getCompanyCode = (storeHash) => {
   }
 };
 
+const checkIsExempted = (data) => {
+  if (data.customer.taxability_code) {
```

Comment:
there are cases when the taxability code is an empty string. we need to also handle that case @prajaktaT-webonise